<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кодирование и сжатие данных</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Декодирование</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QLabel" name="probab_path">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="4">
     <widget class="QLabel" name="dst_path">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QPushButton" name="pushButton_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Выбрать...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="pushButton_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Выбрать...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Выбрать...</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>111</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Входная информация</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="6">
     <widget class="QTextBrowser" name="dsttext"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>81</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Выходной файл</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="QLabel" name="src_path">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>91</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Входной файл</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="4">
     <widget class="QLabel" name="label_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Кодирование</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="6">
     <widget class="QTextBrowser" name="srctext"/>
    </item>
    <item row="4" column="0" colspan="6" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="pushButton_4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Кодировать</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Файл с вероятностями</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="6">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="11" column="0" colspan="6">
     <widget class="QTextBrowser" name="dcdtext"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
